src/main/java/duke/command/CommandNewTask.java

```diff
@@ -22,6 +23,12 @@ public CommandNewTask(String userInput) {
 
     @Override
     public void execute(TaskList taskList) {
+        try {
+            checkS(this.userInput);
+        } catch (DukeException e) {
+            e.printStackTrace();
```

Comment:
We probably should print out the stack trace for the user. Try redirecting it into a log file for future reference instead.